### DISCUSSION TITLE
[QUESTION] CGAL with QT5 support in Embarcadero C++ Builder

### DISCUSSION THREAD

**Discussion by MVoloshin** [CGAL/cgal#6305] [Q&A] [answered]
## Issue Details

Hello, I'd like to use CGAL with QT GUI inside Embarcadero C++ builder.
Does that mean that I'll have to recompile QT5 library with bcc64 too?
At the moment I use CGAL with no CGAL_USE_BASIC_VIEWER and in header-only mode.
Please, tell me how could I add QT GUI support and how complicated it would be.
I found QT5 builds only for MSVC and MinGW, then I know that bcc64 name mangling
is different even from mingw-gcc (64-bit).
Does that imply that I have to build QT5 from scratch using bcc64 too?

## Environment

* Operating system (Windows/Mac/Linux, 32/64 bits): Windows 7 (x64)
* Compiler: bcc64 (LLVM 5.0.2)
* Release or debug mode: Release
* Specific flags used (if any):
* CGAL version: 5.4
* Boost version: 1.70
* Other libraries versions if used (Eigen, TBB, etc.): Eigen

**Comment by lrineau** [comment, marked as answer]
Honestly, your question is way out of our scope, and I doubt any CGAL developer will able to provide an answer. You should consult with developers of Embarcadero C++ builder, for such a question.

I suggest you remove CGAL from the question: as CGAL is header-only (except for its dependencies), you should try first to build a C++ Qt5 program, without CGAL, and try and see if it compiles and runs well with bcc64. As bcc64 seems based on the LLVM clang compiler, I think you should try first using a version of Qt5 built for MSVC.
